creator-node/src/components/tracks/tracksComponentService.js

```diff
@@ -61,6 +64,80 @@ const handleTrackContentRoute = async ({ logContext }, requestProps) => {
   return resp
 }
 
+async function handleTranscodeAndSegment(
+  { logContext },
+  { fileName, fileDir }
+) {
+  return TrackContentUploadManager.transcodeAndSegment(
+    { logContext },
+    { fileName, fileDir }
+  )
+}
+
+async function handleTrackHandOff({ logContext }, requestProps) {
```

Comment:
as discussed this can go away - 
```
const primaryCanHandleTranscode = utils.primaryShouldTranscode() // unit test this - mock is initialized, is queue available
if (primaryCanHandleTranscode) {
// existing logic - already tested
} else {
// wrap this in a simple fn - handoffTranscode(track) which calls asyncProcessingQueue.blahblah
}
```

Reply:
```js
// delegate to TranscodeHandoffMgr
// TranscodeHandoffMgr -> input = track to handoff, return = segments to save - another part/REUSE same logic which already handles saving for the existing flow should still handle the saving
```